src/lib/holocene/table/paginated-table/index.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
-  import ProgressBar from '$lib/holocene/progress-bar.svelte';
   import SkeletonTable from '$lib/holocene/skeleton/table.svelte';
+  import Table from '$lib/holocene/table/table.svelte';
 
   type Item = $$Generic;
 
   export let visibleItems: Item[];
-  export let variant: 'primary' | 'split' = 'primary';
```

Comment:
@Alex-Tideman do you know if this variant is still used anywhere or if it's safe to remove? I searched both oss and cloud and couldn't find any instances where we use `"split"`.

Reply:
Yup 🗑️ it